app/common.php

```diff
@@ -14,12 +14,12 @@
     define('DB_NAME', ltrim($dbopts["path"], '/'));
 }
 
-if (file_exists('config.php')) {
-    require 'config.php';
+if (file_exists(__DIR__.DIRECTORY_SEPARATOR.'config.php')) {
+    require __DIR__.DIRECTORY_SEPARATOR.'config.php';
```

Comment:
I think I started there, but __DIR__ was coming back as being /path/to/kanboard/app.  That's why I included the "..".  Remember, the change is being made to app/common.php, in which __DIR__ would be app.